### PR TITLE
Added IndexedDB iOS WebView support notes

### PR DESCRIPTION
**Pull request by IgorVNC**
### Platforms affected

Cordova iOS

### Motivation and Context
Cordova IndexedDB documentation is outdated

### Description

I relized that IndedxedDB is supported well on iOS 14, hence need to update Codova docs 

### Testing

I have a project with offline capabilities support. 
Normally we use SQLite, but recently started mooving onto IndexedDB.
I did some simple tests and looks like IndexedDB is supported on iOS14 WebView well

### Checklist

not required

### CODE REVIEW

**Comment by IgorVNC** [comment]
Eiger to check under iOS 13 as well, but I do not have iOS device with iOS13

File: www/docs/en/dev/cordova/storage/storage.md [hunk 1]
```diff
@@ -166,6 +166,7 @@ IndexedDB is supported by the underlying WebView on the following Cordova platfo
 
 - Windows (with some limitations)
 - Android (4.4 and above)
+- iOS (14 and above)
```
Comment (timbru31) [inline www/docs/en/dev/cordova/storage/storage.md:169]: This is simply wrong.

**Review by timbru31 — CHANGES_REQUESTED** [review]
Thanks a lot for your PR, however we already have #1042 which we should aim to merge in instead.

File: www/docs/en/dev/cordova/storage/storage.md (hunk 1 above)
Comment (IgorVNC) [reply www/docs/en/dev/cordova/storage/storage.md:169]: ok thanks for the clarification. Then you guys should merge it ASAP cause it produces so much confusion in terms of app architecture decisions

**Comment by timbru31** [comment]
The mentioned PR was just merged 🎉

**Pull request closed without merging**